app/gui2/e2e/locate.ts

```diff
@@ -129,9 +129,13 @@ function componentLocator<T extends string>(className: SanitizeClassName<T>) {
 
 export const graphEditor = componentLocator('GraphEditor')
 export const graphNode = componentLocator('GraphNode')
+export function graphNodeByBinding(page: Locator | Page, binding: string) {
+  return graphNode(page).filter({ has: page.locator('.binding').and(page.getByText(binding)) })
+}
 // @ts-expect-error
 export const anyVisualization = componentLocator('GraphVisualization > *')
 export const circularMenu = componentLocator('CircularMenu')
+export const addNewNodeButton = componentLocator('circle-plus')
```

Comment:
perhaps we should add a `PascalCase` class name matching the component name, for consistency